ts/features/bonus/siciliaVola/store/actions/voucherList.ts

```diff
@@ -31,7 +46,30 @@ export const svVoucherRevocation = createAsyncAction(
   "SV_VOUCHER_REVOCATION_GET_FAILURE"
 )<SvVoucherId, void, NetworkError>();
 
+/**
+ * The user chooses the filter to make the voucher research
+ */
+export const svSetFilter = createStandardAction("SV_SET_FILTER")<FilterState>();
+
+/**
+ * The user reset the filters
+ */
+export const svResetFilter = createStandardAction("SV_RESET_FILTER")<
+  FilterState
+>();
```

Comment:
Yes, I created it because I like to explicit what I want to do. But it's ok also to use only one action.
Removed in [af49a0b](https://github.com/pagopa/io-app/commit/af49a0b13293c56e3524326e77f213d2412a3504)